doc/roadmap.rst

```diff
@@ -67,17 +67,21 @@ following steps:
     analysis functions will be moved from the HNN repository into the ``hnn_core.analysis``.
     module.
 -   Parallelization of the simulation in the ``hnn_core.simulation`` module via a
-    combination of MPI and Joblib.
+    combination of MPI and Joblib. xx reference github issue and milestone here XX
```

Comment:
https://github.com/jonescompneurolab/hnn-core/issues?q=is%3Aissue+is%3Aopen+label%3Aparallelization